backend/src/index.js

```diff
@@ -38,17 +38,15 @@ io.on( 'connection', socket => {
 		}
 
 		if ( !issueUsers.hasOwnProperty( message.user.login ) ) {
-			issueUsers[ message.user.login ] = { sockets: [], tabs: {}, joinedAt: timestamp };
+			issueUsers[ message.user.login ] = { tabs: {}, joinedAt: timestamp };
 		}
 
 		const issueUser = issueUsers[ message.user.login ];
 
-		if ( !issueUser.sockets.includes( socket.id ) ) {
+		if ( !Object.keys( issueUser.tabs ).includes( socket.id ) ) {
```

Comment:
Can the `socket.id` be included twice? I thought it's unique.

Reply:
`socket.id` is a unique value. `!issueUser.tabs.hasOwnProperty( socket.id )` is much simplier condition.